nin.json

```diff
@@ -9,8 +9,8 @@
     "sigveseb",
     "stiaje"
   ],
-  "description": "Glorious Ninjadev Revision Submission",
-  "previewImage": "https://raw.githubusercontent.com/ninjadev/nin/master/nin/frontend/app/images/nin-dark.png",
+  "description": "Ninjadev entry for Revision 2017",
+  "previewImage": "https://github.com/ninjadev/re/blob/master/whatareyousyncingabout.png",
```

Comment:
If we need a direct link to the image, https://raw.githubusercontent.com/ninjadev/re/master/whatareyousyncingabout.png is better